build/nuget/uno.winui.common.targets

```diff
@@ -221,12 +221,8 @@
 	<Target Name="_ValidateIOSStaticRegistrar"
 			BeforeTargets="BeforeBuild"
 			Condition="
-			'$(UnoDisableValidateIOSStaticRegistrar)'==''
-			and (
-				('$(TargetFramework)'=='net6.0-ios' and '$(RuntimeIdentifier)'=='iossimulator-x64')
-				or ('$(ProjectTypeGuids)'=='{FEACFBD2-3405-455C-9665-78FE426C6842};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}' and '$(Platform)'=='iPhoneSimulator')
-			)
-			and (
+			'$(UnoDisableValidateIOSStaticRegistrar)'=='' and
+			'$(TargetFramework)'=='net6.0-ios' and '$(RuntimeIdentifier)'=='iossimulator-x64' and (
```

Comment:
This is always false, but I kept it for now. We will need to update it if the error produced by this target is still relevant.